validator/src/main/resources/expected-data-template/python/eks/rds-mysql-trace.mustache

```diff
@@ -0,0 +1,94 @@
+[
+  {
+    "name": "^{{serviceName}}$",
+    "origin": "^AWS::EKS::Container$",
+    "subsegments": [
+      {
+        "name": "^mysql$",
+        "namespace": "^remote$",
+        "aws": {
+          "account_id": "^{{accountId}}$",
+          "xray": {
+            "auto_instrumentation": true,
+            "sdk": "^opentelemetry for python$"
+          }
+        },
+        "sql": {
+          "connection_string": "^localhost/information_schema$",
+          "url": "^SELECT$",
+          "sanitized_query": "^SELECT \\* FROM tables LIMIT 1;$",
+          "database_type": "^mysql$"
+        },
+        "metadata": {
+          "default": {
+            "aws.span.kind": "^CLIENT$",
+            "EKS.Cluster": "^e2e-python-canary-test$",
+            "PlatformType": "^AWS::EKS$"
+          }
+        },
+        "annotations": {
+          "aws.local.service": "^{{serviceName}}$",
+          "aws.local.operation": "^GET mysql$",
+          "aws.remote.resource.type": "^DB::Connection$",
+          "aws.remote.resource.identifier": "^{{remoteResourceIdentifier}}$",
+          "aws.remote.service": "^mysql$",
+          "aws.remote.operation": "^SELECT$"
+        }
+      }
+    ],
+    "http": {
+      "request": {
+        "url": "^{{endpoint}}/mysql$",
+        "method": "^GET$"
+      },
+      "response": {
+        "status": 200
+      }
+    },
+    "aws": {
+      "account_id": "^{{accountId}}$",
+      "xray": {
+        "auto_instrumentation": true,
+        "sdk": "^opentelemetry for python$"
+      }
+    },
+    "metadata": {
+      "default": {
+        "otel.resource.aws.log.group.names": "^/aws/containerinsights/e2e-python-canary-test/application$",
+        "otel.resource.service.name": "^{{serviceName}}$",
+        "aws.span.kind": "^LOCAL_ROOT$",
+        "otel.resource.cloud.platform": "^aws_eks$",
+        "EKS.Cluster": "^e2e-python-canary-test$",
+        "otel.resource.telemetry.sdk.name": "^opentelemetry$",
+        "otel.resource.ec2.tag.kubernetes.io/cluster/e2e-python-canary-test": "^owned$",
+        "otel.resource.k8s.container.name": "^back-end$",
+        "otel.resource.cloud.account.id": "^{{accountId}}$",
+        "http.route": "^mysql$",
+        "PlatformType": "^AWS::EKS$",
+        "otel.resource.telemetry.sdk.language": "^python$",
+        "otel.resource.cloud.provider": "^aws$"
+      }
+    },
```

Comment:
Do we care to validate everything here? Let's cut off what's not important/specific to App Signals or this use case.
1. If it's already covered by other test cases, doesn't matter here
2. If it isn't, then we probably didn't care about it anyways

Reply:
We don't care anything other than `span.kind` in the metadata section. `span.kind` is mainly for distinguish between `LocalRoot` and `ClientSpan`.

Reply:
Hey Mahad/Mingjie,

Looks like having these values are important, otherwise data model doesn't match and E2E workflow fails.

Failed E2E Workflow: https://github.com/ektabj/aws-application-signals-test-framework/actions/runs/10386611812/job/28758267680

![image](https://github.com/user-attachments/assets/272ec9a2-b73f-4e55-a6f4-bc7989ec3737)

Reply:
Tried adding `StatusCode` back and removing other fields, still same error.

![image](https://github.com/user-attachments/assets/c0898f4a-1487-47ec-9a3f-797a8227e803)

Reply:
Simplified the template and did the testing again.

Python E2E test workflow for us-east-1: https://github.com/ektabj/aws-application-signals-test-framework/actions/runs/10391312721/job/28773927629